deploy/cloud/provider/sdk-app-demo/ingress.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright © 2022 sealos.
```

Comment:
`provider` dir is new direct ,maybe should add one ReadMe file to give some information, like what used for .